docs/spec2006-tutorial.md

```diff
@@ -0,0 +1,517 @@
+In this script, the ISO file name is CPU2006v1.0.1.iso. 
+The script is available [here](https://github.com/darchr/gem5art/blob/master/docs/disks/spec2006/spec2006.json), and we save the file at disk-image/spec2006/spec2006.json.  
+
+In the root folder of experiment,
+```sh
+cd disk-image/spec2006/
+wget https://raw.githubusercontent.com/darchr/gem5art/master/docs/disks/spec2006/spec2006.json
+```
+To build the disk image,
+```sh
+cd disk-image/
+./packer validate spec2006/spec2006.json # validate the script, including checking the input files
+./packer build spec2006/spec2006.json
+```
+The process should not take more than 40 minutes on a fairly recent machine with a normal internet speed. 
+The disk image will be in disk-image/spec2006/spec2006-image/spec2006.  
+
+**Note:**: Packer will output a VNC port that could be used to inspect the building process. 
+Ubuntu has a built-in VNC viewer, namely Remmina.  
+
+**Note:**: [More about using packer and building disk images](disks.md).  
+
+Now, in launch_spec_experiments.py, we make an Artifact object of the disk image.  
+```python
+disk_image = Artifact.registerArtifact(
+    command = './packer build spec2006/spec2006.json',
+    typ = 'disk image',
+    name = 'spec2006',
+    cwd = 'disk-image/',
+    path = 'disk-image/spec2006/spec2006-image/spec2006',
+    inputs = [packer, experiments_repo, m5_binary,],
+    documentation = 'Ubuntu Server with SPEC 2006 installed, m5 binary installed and root auto login'
+)
+```
+### Compiling Linux Kernel  
+In this step, we will download Linux kernel source code and compile the Linux kernel. 
+The file of interest in this step is the vmlinux file.  
+
+First, we download the Linux kernel source code. 
+The following command will shallow clone the linux stable repository as well as checking out the tag v4.19.83, which contains the code for linux kernel version 4.19.83. 
+This also works well for other version numbers.  
+
+In the root of the experiment folder,  
+```sh
+git clone --branch v4.19.83 --depth 1 https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/
+mv linux linux-4.19.83
+```
+Now, in launch_spec_experiments.py, we make an Artifact object of the Linux stable git repo.
+```python
+linux_repo = Artifact.registerArtifact(
+    command = '''git clone git clone --branch v4.19.83 --depth 1 https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/;
+    mv linux linux-4.19.83''',
+    typ = 'git repo',
+    name = 'linux-4.19.83',
+    path =  'linux-4.19.83',
+    cwd = './',
+    documentation = 'Linux kernel 4.19 source code repo obtained in November'
+)
+```
+Next, we compile the Linux kernel. 
+We will make a folder named linux-configs containing all working linux configs. 
+Working Linux configs and documentations for generating a Linux config for a Linux version could be found [here](https://github.com/darchr/gem5art/tree/master/docs/linux-configs).  
```

Comment:
Good. Can this be a link to the tutorial instead. Specifically, if you could link to the part of the tutorial that has the table of what works and what doesn't!

Reply:
Are we going to have the table in boot tutorial? I couldn't find that.

Reply:
We *will*, right @ayaz91 ?

Reply:
Yeah, I will add that once I have results of all runs

Reply:
Can you make a placeholder table so that I can point to?